tsconfig.json

```diff
@@ -1,11 +1,8 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
-    "outDir": "./lib/cjs",
-    "declaration": true,
-    "noEmitOnError": true,
-    "noImplicitAny": true
-  },
-  "include": ["src/**/*"]
+    "compilerOptions": {
+        "lib": ["es2015", "dom"],
+        "noImplicitAny": true,
+        "noEmit": true
+    },
+    "include": ["src/**/*"]
```

Comment:
you still want to exclude node_modules here right?

Reply:
ah yes good call

Reply:
actually just remove node_modules from exclude above